svelteSource/src/lib/validationOutput/ClaimBlock.svelte

```diff
@@ -44,7 +44,7 @@
       disabled = true;
 
       const csrftoken = getCookie("csrftoken");
-      let url = `/rag_evaluation?text=${text}`;
+      let url = `/rag/eval_BM25?text=${text}`;
```

Comment:
Wrong url, and if its positible you should use parameter as argument in request

base_url = 'https://example.com/api'
params = {
    'param1': 'value1',
    'param2': 'value2',
}
response = requests.get(base_url, params=params)

```suggestion
      let url = `/rag/eval_bm25?text=${text}`;
```

Reply:
this is Stefik's issue, I only rewrited his suggestion into url :D